src/resolve-url.js

```diff
@@ -9,6 +9,11 @@ const resolveUrl = (baseUrl, relativeUrl) => {
     return relativeUrl;
   }
 
+  // if baseUrl is a data URI, ignore it and resolve everything relative to window.location
+  if ((/^data:/).test(baseUrl)) {
+    baseUrl = window.location && window.location.href;
```

Comment:
should we add an `|| ''` to the end here so that baseUrl isn't undefined in node?